nexus/examples/qmcpack/rsqmc_misc/estimators/iron_ldaU_dmc.py

```diff
@@ -0,0 +1,221 @@
+optJ123 = generate_qmcpack(
+    identifier           = 'optJ123',
+    path                 = basepath + 'optJ123',
+    job                  = job(cores=16,threads=4,app='qmcpack'),
+    system               = system,
+    twistnum             = 0,
+    qmc                  = 'opt',
+    J2                   = True,
+    J3                   = True,
+    J3_rcut              = J3_rcut,
+    minmethod            = 'oneshift',
+    warmupsteps          = 10,
+    init_cycles          = 0,
+    cycles               = 10,
+    walkers_per_rank     = 256,
+    samples              = 1e5,
+    blocks               = 100,
+    minwalkers           = 0.4,
+    corrections          = [],
+    dependencies         = [(p2q,'orbitals'), (optJ12,'jastrow')],
+    **qmc_shared
+    )
+
+
+#===== Spin density =====
+from qmcpack_input import spindensity
+sdens = spindensity(
+    dr = (0.05, 0.05, 0.05),   # Bohr units
+    #grid = (100,100,100),  # Alternative to dr, an NxNxN grid can be specified
+    )
+
+#===== Energy density =====
+from qmcpack_input import generate_energydensity
+edens = generate_energydensity(
+    coord = 'cartesian',
+    grid  = (100, 100, 100),
+)
+
+#===== Momentum distribution =====
+from qmcpack_input import momentumdistribution
+mom_dist = momentumdistribution(
+    samples = 40,
+    kmax    = 8.0,
+)
+
+#===== One body density matrix =====
+from qmcpack_input import onebodydensitymatrices, sposet
+nbnd = nscf.input.system.nbnd   # Total number of bands solved in KS-DFT for each spin channel (occupied + virtual)
+# For magnetic systems such as iron, the spin-up and spin-down 1RDMs are distinct.
+# Currently, the 1RDM estimator can only project onto a single specified basis.
+# The estimator below uses the spin-up orbitals as the basis in the 1RDM for both spin channels.
+# To get the proper spin-down 1RDM, a seperate QMC run with spin-down basis (spindataset = 1) is needed at the moment.
+dm_est = onebodydensitymatrices(
+    type          = 'OneBodyDensityMatrices',
+    energy_matrix = False,
+    check_overlap = False,
+    integrator    = 'density',
+    evaluator     = 'matrix',
+    samples       = 10,
+    basis         = sposet(type='bspline', size=nbnd, spindataset=0), # Uses the spin up channel (0) KS-DFT orbitals as the DM basis
+)
+
+# In production runs containing estimators, large numbers (100+) of blocks are preferred for robust statistical analysis.
+# More steps per block than specified here are additionally preferred for computational efficiency due to the I/O cost of estimators.
+# Finally, note that for large numbers of blocks, the stat.h5 file's disk footprint can get considerably large.
+# This is especially true for dense grids, say spindensity with (300x300x300).
+qmc = generate_qmcpack(
+    #skip_submit          = True,
+    identifier           = 'qmc',
+    path                 = basepath + 'qmc',
+    job                  = job(cores=12,threads=4,app='qmcpack'),
```

Comment:
Noticed you are using 12 cores here but 16 elsewhere. Any specific reason?

Reply:
Now changed to 16 as well. I must've copied that section from another example.